README.md

```diff
@@ -377,37 +358,20 @@ jobs:
                 "permission_level": "CAN_VIEW"
               }
             ]
-      # Obtain an AAD token and use it to run the notebook on Databricks
-      # Note: If running on AWS or GCP, you can directly pass your service principal
-      # token via the databricks-host input instead
-      - name: Log into Azure
-        uses: Azure/login@v1
-        with:
-          creds: ${{ secrets.AZURE_PROD_CREDENTIALS }}
-      # Get an AAD token for the service principal,
-      # and store it in the DATABRICKS_TOKEN environment variable for use in subsequent steps.
-      # We set the `resource` parameter to the programmatic ID for Azure Databricks.
-      # See https://docs.microsoft.com/en-us/azure/databricks/dev-tools/api/latest/aad/service-prin-aad-token#--get-an-azure-ad-access-token for details.
-      - name: Generate and save AAD token
-        id: generate-token
-        run: |
-          echo "DATABRICKS_PROD_TOKEN=$(az account get-access-token \
-          --resource=2ff814a6-3304-4ab8-85cb-cd0e6f879c1d \
-          --query accessToken -o tsv)" >> $GITHUB_ENV
       - name: Trigger notebook in prod
         uses: databricks/run-notebook@v0
         with:
-          databricks-host: https://adb-prod.XX.azuredatabricks.net
-          databricks-token: ${{ env.DATABRICKS_PROD_TOKEN }}
+          databricks-host: https://xxx-prod.cloud.databricks.com
+          databricks-token: ${{ secrets.DATABRICKS_PROD_TOKEN }}
           local-notebook-path: notebooks/MainNotebook.py
-          # The cluster JSON below is for Azure Databricks. On AWS and GCP, set
-          # node_type_id to an appropriate node type, e.g. "i3.xlarge" for
-          # AWS or "n1-highmem-4" for GCP
+          # The cluster JSON below is for AWS workspaces. On Azure and GCP, set
+          # node_type_id to an appropriate node type, e.g. "Standard_D3_v2" for
+          # Azure or "n1-highmem-4" for GCP
           new-cluster-json: >
             {
               "num_workers": 1,
               "spark_version": "10.4.x-scala2.12",
-              "node_type_id": "Standard_D3_v2"
+              "node_type_id": "i3.xlarge"
```

Comment:
Why are you setting node_type_id to AWS specific? Should we set these back to Azure specific e.g. `Standard_D3_v2`

Reply:
So that users will not have to generate an AAD token three times.

Reply:
The thing is, AAD tokens only need to be generated once to be used across three workspaces (unless the total job time is over an hour)

Reply:
I was concerned that by demonstrating generating the AAD token three times, users would assume that you needed to generate one for each workspace.